### PR TITLE
Update Generator

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

* Update template files for new repo file structor
* Replace nightwatch template & script with webdriver.io template & script
* Update `.npmignore` file

@cerner/terra

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@ Changelog
 
 Unreleased
 ------------------
+### Added
+* Update template files for new repo file structor
```
Comment (JakeLaCombe) [inline CHANGELOG.md]: *structure

File: generators/app/index.js [hunk 2]
```diff
@@ -214,7 +240,10 @@ module.exports = yeoman.Base.extend({
 
     this.fs.copy(
       this.templatePath('LICENSE'),
-      this.destinationPath(this.props.baseDirectory + 'LICENSE')
+      this.destinationPath(this.props.baseDirectory + 'LICENSE'),
+      {
+        currentYear: this.props.currentYear
```
[the remark this answers is not in the document]
Comment (JakeLaCombe) [reply generators/app/index.js]: Are we ok with this? I left the comment on another code review about this, and @bjankord said he had to consult with James on what year the copyright is valid for.

[the remark this answers is not in the document]
Comment (JakeLaCombe) [reply generators/app/index.js]: Looks like there is already an issue logged for it. https://github.com/cerner/terra-framework/issues/65

File: generators/app/templates/Index.jsx [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
 // Example Files
+import Default<%= projectClassName %> from './<%= importPath %>Default<%= projectClassName %>';
+import Default<%= projectClassName %>Src from '!raw-loader!./<%= importPath %>Default<%= projectClassName %>.jsx';
 
 const <%= projectClassName %>Examples = () => (
   <div>
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
+
+    <h1 style={{ paddingBottom: '0.3em', borderBottom: '1px solid #eaecef' }}>Examples</h1>
```
Comment (JakeLaCombe) [inline generators/app/templates/Index.jsx]: Can we use one of our terra heading elements as part of our generator now? Feels like the best way to get users to use our components is to build our example sites with them. https://terra-core-deployed.herokuapp.com/static/#/site/heading

File: generators/app/templates/package.json [hunk 4]
```diff
@@ -41,8 +41,8 @@
     "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
     "lint:scss": "stylelint src/**/*.scss",
     "props-table": "props-table ./src/<%=jsxFileName%>.jsx --out-dir ./docs/props-table",
-    "test": "npm run test:jest && npm run test:nightwatch",
+    "test": "npm run test:jest && npm run test:wdio",
```
Comment (JakeLaCombe) [inline generators/app/templates/package.json]: Nice! 😄 👍

File: test/app.js [hunk 5]
```diff
@@ -82,20 +84,23 @@ describe('generator-terra-module:app', function () {
         assert.fileContent(readme, `[Documentation](https://github.com/cerner/${repository}/tree/master/packages/${repository}-monster-cookies/docs)`);
       });
 
-      it('fills the examples Index file with project data', function () {
-        const index = `packages/${repository}-site/src/examples/monster-cookies/Index.jsx`;
+      it('fills the site examples Index file with project data', function () {
+        let index = `packages/${repository}-site/src/examples/monster-cookies/Index.jsx`;
+        let fileImportPath = `${repository}-monster-cookies`;
+        if (repositoryName === 'terra-framwork') {
```
Comment (JakeLaCombe) [inline test/app.js]: terra-framework*

File: test/app.js [hunk 6]
```diff
@@ -25,10 +25,12 @@ describe('generator-terra-module:app', function () {
       });
 
       it('creates files', function () {
+        const examplePath = repositoryName === 'terra-framwork' ? '/examples/Index.site-page.jsx' : '-site/src/examples/monster-cookies/Index.jsx';
```
Comment (JakeLaCombe) [inline test/app.js]: terra-framework*

File: generators/app/templates/Index.jsx (hunk 3 above)
Comment (emilyrohrbough) [reply generators/app/templates/Index.jsx]: @JakeLaCombe @bjankord  by using our header we would require every package to add Header as a dependency. Do we want that?

**Pull request merged**